app/uk/gov/hmrc/trackingconsentfrontend/assets/src/common/withContentLoaded.ts

```diff
@@ -0,0 +1,7 @@
+const withContentLoaded = render => (document, ...args) => {
```

Comment:
I think this should have some tests around it.

Reply:
I've added a test. We could consider removing the equivalent tests from the formHandler.